client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -23,6 +23,9 @@ const useSteps = ( { flowName, hasPaidDomain, isDestinationSetupSiteFlow } ) =>
 		case 'domain':
 			steps = [ { title: __( 'Preparing your domain' ) } ];
 			break;
+		case 'site-content-collection':
+			steps = [ { title: __( 'Saving your content' ) } ];
```

Comment:
Shall we duplicate these steps so that there are 2 delayed steps and to segments in the progress bar? Currently the user experience feels a bit broken because the progress bar fills instantly and the user has to wait for sometime in this state.

Reply:
Thanks, that makes sense to me. I have added an additional step, and the processing screen does look better!

https://user-images.githubusercontent.com/5436027/154215261-04f64997-1733-4269-94ad-9e5218dd93f5.mp4